specs/osquerybeat.spec.yml

```diff
@@ -13,6 +13,7 @@ inputs:
     outputs:
       - elasticsearch
       - logstash
+      - kafka
```

Comment:
@aleksmaus will osquerybeat support Kafka without code changes through libbeat, like the other Beats?

Reply:
Looks like it does, it uses the standard root command that will eventually call the beat `makeOutputFactory` function https://github.com/elastic/beats/blob/c7d59203f2fd7d0a284688ee4c395cb64249f375/libbeat/cmd/instance/beat.go#L377

Reply:
yeah, osquerybeat relies on libbeat 100%, should work